user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Configuring_components/Customize_Component_Layout.md

```diff
@@ -28,7 +28,17 @@ Each component in a dashboard or low-code app has a number of default options. B
 
      - In the *Colors* section, specify a custom background color and/or font color, either by specifying the color in RGB format or by using the color picker box on the right.
 
-       From DataMiner 10.0.12 onwards, under *Colors* > *Color palette*, you can customize additional component colors, e.g. for the lines in a line chart.
+       Under *Colors* > *Data colors* or *Colors* > *Color palette* (prior to DataMiner 10.3.0 [CU16]/10.4.0 [CU4]/10.4.7<!--RN 39739-->), you can customize additional component colors, e.g. for the lines in a line chart.
+
+       From DataMiner 10.3.0 [CU16]/10.4.0 [CU4]/10.4.7 onwards<!--RN 39739-->, under *Colors* > *Data colors*, you can specify conditional colors. If a display label of the data matches the specified text, that data will inherit the conditional color.
+
+       To add a conditional color:
+
+       1. Select *Add conditional color*.
+
+       1. Enter the text to match the display label. Partial matches are allowed. For example, entering `Tot` will match data labeled `Total`.
+
+       1. Choose a custom color, either by specifying the color in RGB format or by using the color picker box on the right.
```

Comment:
RGB isn't the only accepted format, you can also use the hex or names: https://www.w3schools.com/tags/ref_colornames.asp

Reply:
```suggestion
       1. Choose a custom color, either by specifying the color in RGB format, by entering the hex value or HTML color name, or by using the color picker box on the right.
```